docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3'
```

Comment:
Could you please move this file into the `/database` dir? Currently it is being used exclusively for database setup. Thanks!

Reply:
Please also adjust the version as seen here: https://docs.docker.com/compose/compose-file/compose-versioning/#version-3

Currently it's 3.0, but there is already 3.8.

Reply:
Updated the location & version of the docker file.